src/Amadeus/Client/Struct/Service/IntegratedPricing/PricingOptionKey.php

```diff
@@ -30,6 +30,7 @@
  */
 class PricingOptionKey
 {
+	const OPTION_FARE_BASIS_SIMPLE_OVERRIDE = "FBA";
```

Comment:
intendation is wrong

Reply:
I made some change, so key I am not using from this file. Instead directly taking from request object. So removing my changes from this file